README.md

```diff
@@ -1,92 +1,34 @@
 # scala-baseimage
 
-[![](http://badge-imagelayers.iron.io/broadinstitute/scala-baseimage:latest.svg)](http://imagelayers.iron.io/?images=broadinstitute/scala-baseimage:latest 'Get your own badge on imagelayers.iron.io')
+This is a base Docker image for Scala applications that need to run commands with sbt. Useful for development, building, and testing. **Compiled applications shouldn't need this.**
 
-This is a base Docker image for Scala applications, specifically web services.  Currently this image will have installed:
+Contains:
+* AdoptOpenJDK Java 11
+* Scala
+* SBT
+* Git
 
-* Java 8
-* Scala 2.11.7
-* SBT 0.13.9
+This image is built off of the latest AdoptOpenJDK Java 11 image. It is currently published manually.
 
-This image is build off of [baseimage-docker](http://phusion.github.io/baseimage-docker/), which provides an init system and process supervisor called [runit](http://smarden.org/runit/) for starting and monitoring services.
+The current version of the image is here: [broadinstitute/scala-baseimage](https://registry.hub.docker.com/r/broadinstitute/scala-baseimage/)
 
-This image is available on Dockerhub ([here](https://registry.hub.docker.com/u/broadinstitute/scala-baseimage/))
+***N.B. The `latest` tag points to a legacy Java 8 image!***
 
-## Usage
+### Available tags
 
-It is recommended that the `Dockerfile` be located in the root of the project source directory.  This way, the `ADD . /cromwell` line below is a good convention for adding the project source code to the Docker container.
+Java 11:
 
-Within the project's source tree are two Docker-related files:
+| Scala   | SBT   | Tag                   |
+|---------|-------|-----------------------|
+| 2.12.11 | 1.4.7 | `jdk11-2.12.11-1.4.7` |
+| 2.12.12 | 1.4.6 | `jdk11-2.12.12-1.4.6` |
```

Comment:
is it intentional that the later version of scala uses the earlier version of sbt, and the later version of sbt uses the earlier version of scala?

Reply:
The pairings match rawls (with the update PR I'm about to publish) and orch.